build_restart.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -e
+# ビルド
+xcodebuild -workspace macSKK.xcodeproj/project.xcworkspace -scheme macSKK clean archive -archivePath build/archive.xcarchive
+# 上書き
+sudo rm -rf /Library/Input\ Methods/macSKK.app
```

Comment:
macSKKは基本ユーザーのホームディレクトリに入れています。管理者権限が要求されていたのはもしかしてここかも?

```suggestion
rm -rf ~/Library/Input\ Methods/macSKK.app
```

Reply:
あ、失礼 🙏 たしかにここもともとは`/Library/Input\ Methods/`に入れようとしていて、ところがmacSKKはホームディレクトリ以下の`~/Library`だったので消したけど`sudo`消しわすれてました 🙇